source/Crew.cpp

```diff
@@ -12,8 +12,9 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 #include "Crew.h"
 #include "Files.h"
+#include "MoraleAffected.h"
+#include "MoraleDescription.h"
 #include "GameData.h"
-#include "PlayerInfo.h"
```

Comment:
I found it necessary to remove this dependency because I had an `include` loop across a few files.